manage.py

```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# 
-# 
-# 
-# Copyright (c) 2009-2013 University of Dundee. 
```

Comment:
Considering the repos were split, it makes some sense to keep the old ones, but I leave that to you. If everyone at this point wants to say "good enough", no worries.